newIDE/app/src/EventsFunctionsExtensionsLoader/Storage/LocalEventsFunctionsExtensionWriter.js

```diff
@@ -62,4 +62,45 @@ export default class LocalEventsFunctionsExtensionWriter {
       throw err;
     });
   };
+
+  static chooseCustomObjectFile = (objectName?: string): Promise<?string> => {
```

Comment:
Is it ok to add this here?
It could be renamed `LocalProjectPartExporter`, but there is probably a better name.